package.json

```diff
@@ -18,11 +18,19 @@
     },
     "homepage": "https://github.com/NJaryal/ReactApp#readme",
     "dependencies": {
+        "@babel/polyfill": "^7.4.4",
```

Comment:
it's dev deph

Reply:
FYI
Because this is a polyfill (which will run before your source code), we need it to be a dependency, not a devDependency

https://babeljs.io/docs/en/next/babel-polyfill.html

So i included inside dependencies.